.circleci/config.yml

```diff
@@ -43,10 +43,19 @@ jobs:
           command: $GRADLEW checkstyle
       - run:
           name: ktlint
-          command: $GRADLEW ktlint
+          command: $GRADLEW ciktlint
       - run:
           name: Lint
           command: $GRADLEW lintVanillaRelease || (grep -A20 -B2 'severity="Error"' -r --include="*.xml" WordPress libs; exit 1)
+      - run:
+          name: Violations
+          when: on_fail
+          command: |
+            if [ -n "$GITHUB_API_TOKEN" ]; then
+              $GRADLEW violationCommentsToGitHub -DGITHUB_PULLREQUESTID=${CIRCLE_PULL_REQUEST##*/} -DGITHUB_OAUTH2TOKEN=$GITHUB_API_TOKEN
+            else
+              echo "Not running danger because $GITHUB_API_TOKEN is not found"
```

Comment:
Maybe change it to something like `Not posting lint errors to Github because ...`?

Also, you probably want to escape the `$` here. so it will looks like this:

```
echo "Not running danger because \$GITHUB_API_TOKEN is not found"
```

Otherwise, it will print `Not running danger because  is not found`